packages/notifier/src/storesub.js

```diff
@@ -117,3 +187,31 @@ export const makeStoredPublisherKit = (storageNode, marshaller, childPath) => {
     subscriber,
   };
 };
+
+/**
+ * Like makePublishKit this makes a `{ publisher, subscriber }` pair for doing efficient
+ * distributed pub/sub supporting both "each" and "latest" iteration
+ * of published values.
+ *
+ * What's different is `subscriber` tees records, writing out to storageNode.
+ *
+ * @template [T=unknown]
+ * @param {ERef<StorageNode>} storageNode
+ * @param {ERef<Marshaller>} marshaller
+ * @param {string} [childPath]
+ * @returns {StoredPublishKit<T>}
+ */
+export const makeStoredPublishKit = (storageNode, marshaller, childPath) => {
```

Comment:
I remain opposed to this `childPath` argument; it's just out of scope for the notifier package.

Reply:
@gibson042 any suggestion for what to do instead?

Reply:
We have a `makeChildNode` method somewhere; is that a different package? Does this function belong there?

Reply:
@gibson042 you get to decide what's out of scope for `notifier` package so I'll find another place.

By the same token, should `makeStoredPublishKit` and `StoredPublishKit` also go? They can be made from `makePublishKit` and `makeStoredSubscriber`. I think the answer is no unless we also move out makeStoredSubscriber`. Though I wonder if we shouldn't export `makeStoredSubscriber` and only export `makeStoredPublishKit`.

Reply:
I'm on the fence about "makeStored" helpers being in the notifier package, but am willing to run with them for now. However, this parameter in particular just seems too tightly coupled... I'd strongly prefer invocations like `makeStoredPublisherKit(E(storageNode).getChildNode(childNodeName), marshaller)` over `makeStoredPublisherKit(storageNode, marshaller, childNodeName)`.